driver/opoptions/channel.go

```diff
@@ -41,6 +41,38 @@ func WithEager() util.Option {
 	}
 }
 
+// WithFuzzyMatchInput makes the channel use fuzzy matching when reading until the input -- this can
+// be useful for inputs that devices split across multiple lines, for example, when entering a b64
+// key in nokia srlinux things look something like this:
```

Comment:
I don't think here we use base64, but rather a raw text input.

Reply:
yea good call, fixing to just say "cert key"